assets/aml-benchmark/components/batch-benchmark-inference/spec.yaml

```diff
@@ -132,9 +136,10 @@ jobs:
   # Preparer
   batch_inference_preparer: 
     type: command
-    component: azureml:batch_inference_preparer:0.0.2
+    component: ../batch-inference-preparer/spec.yaml
```

Comment:
keep this with the new  version `component: azureml:batch_inference_preparer:0.0.<nextversion>`

Reply:
For other components, you also need to bump the version